docs/fr/README-FR.md

```diff
@@ -0,0 +1,147 @@
+<!-- Deep Learning (with PyTorch)
+-->
+
+# Apprentissage profond (avec PyTorch) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Atcold/pytorch-Deep-Learning/master)
+
+
+
+<!-- This notebook repository now has a [companion website](https://atcold.github.io/pytorch-Deep-Learning/), where all the course material can be found in video and textual format.
+-->
+
+Ce répertoire a maintenant un [site web associé](https://atcold.github.io/pytorch-Deep-Learning/), où tout le matériel de cours peut être trouvé au format vidéo et textuel.
+
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic -->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md)
+[FR](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/fr/README-FR.md)
```

Comment:
Use 🇫🇷    please and `&nbsp;` like for the other flags.

Reply:
Done. I also add arabic path for some file where it was missing